src/auth/auth0-plugin.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import Vue from "vue";
-import createAuth0Client from "@auth0/auth0-spa-js";
+import { createAuth0Client } from "@auth0/auth0-spa-js";
```

Comment:
We dropped the default export, see https://github.com/auth0/auth0-spa-js/blob/master/MIGRATION_GUIDE.md#no-more-default-export